dataactvalidator/interfaces/validatorStagingInterface.py

```diff
@@ -1,77 +1,52 @@
 from dataactcore.models.stagingInterface import StagingInterface
-from dataactcore.models.stagingModels import FieldNameMap
-from dataactcore.utils.responseException import ResponseException
-from dataactcore.utils.statusCode import StatusCode
-from dataactvalidator.interfaces.validatorJobTrackerInterface import ValidatorJobTrackerInterface
-from sqlalchemy import MetaData, Table
-from sqlalchemy.exc import NoSuchTableError
+from dataactcore.models.stagingModels import Appropriation, ObjectClassProgramActivity, AwardFinancial, AwardFinancialAssistance
 
 
 class ValidatorStagingInterface(StagingInterface):
     """ Manages all interaction with the staging database """
 
-    def dropTable(self,table):
+    def insertSubmissionRecordByFileType(self, record, fileType):
+        """Insert a submitted file row into staging.
+
+        Args:
+            record: record to insert (Dict)
+            submissionId: submissionId associated with this record (int)
+            fileType: originating file type of the record (string)
         """
+        if fileType == "award":
+            rec = AwardFinancialAssistance(**record)
+        elif fileType == "award_financial":
+            rec = AwardFinancial(**record)
+        elif fileType == "appropriations":
+            rec = Appropriation(**record)
+        elif fileType == "program_activity":
+            rec = ObjectClassProgramActivity(**record)
+        self.session.add(rec)
+        self.session.commit()
```

Comment:
Now that we've had the SQL validation convo, what do you think about moving the commit here, so we commit to staging before the final batch of error records gets written? Then we could slot in the call to the SQL validator between these two lines.

If this is going to require more thought, I'll leave things as is, and we can change when working on the SQL stories.